pkg/host/machine_info_linux.go

```diff
@@ -13,6 +13,8 @@ import (
 	"regexp"
 	"strconv"
 	"strings"
+
+	"github.com/gobwas/glob"
```

Comment:
External dependencies are a liability. Addition of every external dependency should be weighted.
Here it does not look like it's worth it. This library does not even implement the same pattern rules as we use for positive matches. This looks like a bad asymmetry.
I suggest we use the same filepath.Glob for negative matches and then subtract one set from another (will probably be easier if we store both sets in map's right away, will also get rid of the separate dedup step).
This will be symmetric for positive/negative rules and won't require the dependency.

Reply:
Or maybe we should use a regexp for negative?
It won't require scanning file tree multiple times and is as powerful as powerful as https://github.com/gobwas/glob.